python/tank/bootstrap/manager.py

```diff
@@ -438,7 +592,10 @@ def _cache_apps(self, tk, do_post_install=False):
         for idx, descriptor in enumerate(descriptors):
 
             if not descriptor.exists_local():
-                self._report_progress("Downloading %s..." % descriptor, idx, len(descriptors))
+                self._report_progress(progress_value=0.4 + idx*(0.3/len(descriptors)),
```

Comment:
Some magic numbers here. Maybe a quick comment to explain what the calculation is doing here?